python/lsst/obs/subaru/strayLight/waveletCompression.py

```diff
@@ -14,24 +14,22 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-#----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
```

Comment:
I know this is picky and feel free to ignore it, but consider getting rid of the line?

Reply:
Yes. Sorry. I should have removed it.